config/deployment.yml

```diff
@@ -15,6 +15,10 @@ spec:
   revisionHistoryLimit: 0
   template:
     metadata:
+      annotations:
+        #! Necessary at least on GKE (which includes a default apparmor policy)
+        #! which seems to prevent execution of mount syscall even within a user namespace
+        container.apparmor.security.beta.kubernetes.io/kapp-controller-sidecarexec: unconfined
```

Comment:
i did run it there already, though i need to rerun after latest changes to double check.

Reply:
I guess I don't see it marked as "checked" off - so I wasn't sure where you were at in your testing

<img width="1354" alt="Screenshot 2022-07-15 at 12 43 42" src="https://user-images.githubusercontent.com/6075668/179290533-a7620483-1953-43d9-9494-10945134316b.png">

Reply:
i only checked off minikube since ive been actively developing against it. ill be checking off other things as i try to run against them. (if anybody else wants to take it for a spin on any env, just edit the description marking in progress?).